server/src/test/java/org/elasticsearch/snapshots/SnapshotResiliencyTests.java

```diff
@@ -544,8 +550,37 @@ private void assertNoStaleRepositoryData() throws IOException {
         }
     }
 
+    // Lucene's mock file system randomly generates empty `extra0` files that break the deletion of blob-store directories.
+    // We clean those up here before checking a blob-store for stale files in this test.
```

Comment:
Can we make an adjustment to "stale files in this test", because it's not a test, but just a private method. Probably just remove "in this test".

Reply:
sure removed the "in this test", thanks for taking a look!